src/client/src/components/ProjectName/index.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
+import { useDispatch, useSelector } from "react-redux";
```

Comment:
Project Name ya no pone myApp por defecto
Save To no pone ningún path por defecto
Cuando haces click en browse da error: invalid command used

![image](https://user-images.githubusercontent.com/1333036/91410606-7155ec80-e847-11ea-9bd1-0fe387b7788d.png)